### DISCUSSION TITLE
Beta (+,-) and internal bremsstrahlung spectra request

### DISCUSSION THREAD

**Discussion by gakabani** [radioactivedecay/radioactivedecay#110] [Q&A]
Dear Friends,

Is there a method or attribute in the radioactive decay package that generates the beta spectrum of radionuclides? 
I used to run RADLST and generate the beta spectrum (+,-) and internal bremsstrahlung.
Thanks for all your help.

**Comment by alexmalins** [comment]
it's not available in this package currently, no

there's some more information in #46 which you also found
